### PR TITLE
Add a few more ??=

### PR DESCRIPTION
**Pull request by tigitz**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | -
| License       | MIT
| Doc PR        | -

Continuation of https://github.com/symfony/symfony/pull/47549 using rector with this config

```php
<?php

declare(strict_types=1);

use Rector\Config\RectorConfig;
use Rector\Php74\Rector\Assign\NullCoalescingOperatorRector;

return static function (RectorConfig $rectorConfig): void {
   $rectorConfig->rule(NullCoalescingOperatorRector::class);
};
```

### CODE REVIEW

**Comment by tigitz** [comment]
friendly ping @nicolas-grekas as it's a continuation of your work

**Review by stof — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

File: src/Symfony/Component/Cache/Adapter/AbstractAdapter.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public static function createSystemCache(string $namespace, int $defaultLifetime
             $opcache->setLogger($logger);
         }
 
-        if (!self::$apcuSupported = self::$apcuSupported ?? ApcuAdapter::isSupported()) {
+        if (!(self::$apcuSupported ??= ApcuAdapter::isSupported())) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Cache/Adapter/AbstractAdapter.php]: no need for the extra brackets

**Review by nicolas-grekas — APPROVED** [review]
just one minor thing

**Review by derrabus — APPROVED** [review]

**Comment by tigitz** [comment]
@nicolas-grekas Done :ok_hand: 

Thanks everyone for the review.

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @tigitz.

**Pull request merged**